openslides/core/static/templates/index.html

```diff
@@ -146,16 +179,17 @@ <h3 class="modal-title" translate>Please sign in!</h3>
           <div ui-view></div>
         </div>
       </div>
-      <hr />
-      <footer>
-        <small>
-          &copy; Copyright 2011-2015 |
-          Powered by <a href="http://openslides.org" target="_blank">OpenSlides</a> |
-          <a ui-sref="version">Version</a>
-        </small>
-      </footer>
     </div><!--/#content-->
   </div><!--/.row-->
+
+  <hr>
+  <footer>
+    <small>
+      &copy; Copyright 2011-2015 |
```

Comment:
I never understood why we need the years. I am against it, because it has no use and we have to change it every year